devel/lab/Portfile

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        zaquestion lab 0.14.0 v
+categories          devel
+platforms           darwin
+license             The Unlicense
```

Comment:
I don't think this is a valid license in our list. (The license field isn't freeform; it's used by our automation to determine what can be binary redistributed.)

Reply:
I've honestly never seen that license either. It's not my code, I saw this as a good opportunity to get my feet wet contributing to MacPorts when I noticed there wasn't a package for it.

Reply:
What do you recommend for license, then? I can't seem to find the list you refer to.

The [license](https://github.com/zaquestion/lab/blob/master/LICENSE) is descriped as rather permissive.

Reply:
I found the list. Considering Unlicense opens with

> A license with no conditions whatsoever which dedicates works to the public domain.

I'll set it to "public-domain"

Reply:
It would be helpful if lint checked for valid license against that list, methinks. Should that be a TRAC ticket suggestion?